### PR TITLE
Add function details in external attacher exported functions.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -104,6 +105,7 @@ func NewCSIAttachController(client kubernetes.Interface, attacherName string, ha
 	return ctrl
 }
 
+// Run start CSI attacher and listens on channel events
```
Comment (jsafrane) [inline pkg/controller/controller.go]: start -> starts

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -38,6 +38,7 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
+// CSIAttachController contain required interfaces of handlers, eventrecorder, workqueue and listers.
```
Comment (jsafrane) [inline pkg/controller/controller.go]: contain -> contains

And IMO it's not interesting what *exactly* it contains (handlers, ...), but what is it good for, so "CSIAttachController is a controller that attaches / detaches CSI volumes using provided Handler interface." Or something else in this direction.

File: pkg/controller/util.go [hunk 3]
```diff
@@ -90,6 +90,7 @@ const (
 	vaNodeIDAnnotation         = "csi.alpha.kubernetes.io/node-id"
 )
 
+// SanitizeDriverName sanitize provided driver name.
```
Comment (jsafrane) [inline pkg/controller/util.go]: sanitize -> sanitizes

File: pkg/controller/controller.go (hunk 2 above)
Comment (humblec) [reply pkg/controller/controller.go]: Sure, Done.

File: pkg/controller/controller.go (hunk 1 above)
Comment (humblec) [reply pkg/controller/controller.go]: Done

File: pkg/controller/util.go (hunk 3 above)
Comment (humblec) [reply pkg/controller/util.go]: Done

**Comment by humblec** [comment]
@jsafrane Thanks! comments are addressed. PTAL .

File: pkg/controller/util.go (hunk 3 above)
Comment (pohly) [reply pkg/controller/util.go]: No offense (I really appreciate your work on cleaning up), but this is the kind of comment one gets when the tooling forces one to add something: the comment doesn't add any information that isn't already in the function name and thus adds no value.

I'm now torn between asking for a better comment (for example, one that explains what "sanitize" means in this context) and just ignoring the comment.

**Comment by jsafrane** [comment]
/lgtm
/approve

Thanks for the PR!

**Pull request merged**